### PR TITLE
[bitnami/harbor] Add configurable session timeout to harbor

### PR DESCRIPTION
**Pull request by holzman**
* [bitnami/harbor] Add configurable session timeout to harbor

The beego backend for harbor has a short (3600s) session timeout by
default, which can be too short for frequent users.  This adds a
configurable core.sessionTimeout value.

Signed-off-by: Burt Holzman <burt@fnal.gov>

* [bitnami/harbor] Rerender README on master

Signed-off-by: Burt Holzman <burt@fnal.gov>

**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/harbor/values.yaml [hunk 1]
```diff
@@ -1217,6 +1217,10 @@ core:
     ## Enable debug mode
     ##
     debug: false
+
+  ## @param core.sessionLifetime Explicitly set a session timeout (in seconds) overriding the backend default.
+  ## sessionLifetime: 86400
```
Comment (miguelaeh) [inline bitnami/harbor/values.yaml:1222]:
```suggestion
  ##
```

File: bitnami/harbor/Chart.yaml [hunk 2]
```diff
@@ -34,4 +34,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-harbor-registry
   - https://github.com/bitnami/bitnami-docker-harbor-registryctl
   - https://goharbor.io/
-version: 12.2.2
+version: 12.2.3
```
Comment (miguelaeh) [inline bitnami/harbor/Chart.yaml:37]:
```suggestion
version: 12.3.0
```

**Review by miguelaeh — CHANGES_REQUESTED** [review]
thanks for the PR!

File: bitnami/harbor/Chart.yaml (hunk 2 above)
Comment (holzman) [reply bitnami/harbor/Chart.yaml]: ok

**Comment by holzman** [comment]
Trivy scan seems to be timing out across the board (https://github.com/bitnami/charts/pulls?q=is%3Apr+is%3Aopen+harbor)

**Review by miguelaeh — APPROVED** [review]
LGTM

**Comment by miguelaeh** [comment]
Could you please solve the conflicts, I am not sure if the jobs got stuck of they don't start because of the conflict

**Comment by holzman** [comment]
Rebased, conflicts resolved.

**Review by miguelaeh — APPROVED** [review]

**Pull request merged**